Task06/decorators.py

```diff
@@ -0,0 +1,30 @@
+import time
+
+def measure_elapsed_time(func):
+    def wrapper(*args, **kwargs):
+        time_start = time.time()
+        result = func(*args, **kwargs)
+        time_end = time.time()
+        print(f"{func.__name__} call took {round(time_end-time_start, 1)} seconds")
```

Comment:
```suggestion
        duration = time.time() - time_start
        print(f"{func.__name__} call took {round(duration, 1)} seconds")
```
Как по мне, так код читается немного лучше